eng/Build.props

```diff
@@ -4,16 +4,26 @@
        they depend on assets from other verticals that are built in the first build pass. -->
   <ItemGroup Condition="'$(DotNetBuildPass)' == '2' and
                         '$(OS)' == 'Windows_NT' and
-                        '$(TargetArchitecture)' == 'x64'">
+                        '$(TargetArchitecture)' == 'x64' and
+                        '$(BuildWorkloads)' != 'true'">
```

Comment:
It might make things cleaner to actually build these components via an opt-in property in the future instead of by default based on the BuildPass + config

Reply:
Yes, I agree